posthog/helpers/session_recording.py

```diff
@@ -63,6 +69,9 @@ def compress_and_chunk_snapshots(events: List[Event], chunk_size=512 * 1024) ->
                 },
             },
         }
+        if window_id:
+            chunk_data["properties"]["$snapshot_data"]["window_id"] = window_id
```

Comment:
Q: Why are we setting $session_id at one layer higher but window_id one layer deeper? Why not keep the two symmetrical?

Reply:
Right now, there's no `properties` column on the `session_recording_events` table. Instead, the `session_id` and `snapshot_data` get extracted out from properties to their own columns, and the rest of the properties are dropped. (https://github.com/PostHog/posthog/blob/master/plugin-server/src/worker/ingestion/process-event.ts#L141)

The cleaner solution here is definitely to add a `window_id` column to the `session_recording_events` table, but I was trying to avoid a migration on the `session_recording_events` (heard that we don't want to be doing those, but I don't really know the cost).

@macobo Would love your thoughts on the trade-off here, and if you think adding the `window_id` column makes sense

Reply:
I think it does make sense to add this as a column :)

It isn't _that_ bad, see ee/clickhouse/migrations/0012_person_id_deleted_column.py for an example and tag #team-platform in the review to make sure no conflicts on cloud